g3/generate/prompts/commit/creator.py

```diff
@@ -5,58 +5,85 @@
-        messages = [
-            {
-                "role": "system",
-                "content": "You are a helpful assistant creating commit messages for git commits in a software "
-                           "engineering team based on their stashed git changes which will be provided to you."
-                           "We defined a good commit message as one that explains what was changed, and if usefull, why"
-                           " the change was made. Don't include the provided code in your response or the words commit "
-                           "message. Be brief on your response using less than 7-10 words, use imperative and start "
-                           "with a verb. The goal is to briefly complete a sentence which starts like 'This commit if "
-                           "applied will:' but dont include this sentence in your response. "
-                           "Whatever follows should be the commit message that you suggest. "
-                           f"Include in your reponse the following: {self.include}. " if self.include else ""
-                           f"Incorporate the following Jira ticket: {self.jira}. " if self.jira else ""
-                           f"Use the following tone when creating the commit message: {self.tone}. " if self.tone else ""
-                           "Your response should be in markdown format."
-
-            },
-            {
-                "role": "user",
-                "content": f"Please provide a commit message for the provided code. Code: ```{sample.get('code')}```"
-            },
-            {
-                "role": "assistant",
-                "content": sample.get("message")
-            },
+        commit_creation_messages = [{
+            "role": "system",
+            "content": "You are a helpful assistant creating commit messages for git commits in a software " \
+                       "engineering team based on their stashed git changes which will be provided to you." \
+                       "We defined a good commit message as one that explains what was changed, and if usefull, why" \
+                       " the change was made. Don't include the provided code in your response or the words commit " \
+                       "message. Be brief on your response using less than 7-10 words, use imperative and start " \
+                       "with a verb. The goal is to briefly complete a sentence which starts like 'This commit if " \
+                       "applied will:' but dont include this sentence in your response. " \
+                       "Whatever follows should be the commit message that you suggest. " \
+                       f"Include in your reponse the following: {include}. " if include else "" \
+                        f"Incorporate the following Jira ticket: {jira}. " if jira else "" \
+                        f"Use the following tone when creating the commit message: {tone}. " if tone else "" \
```

Comment:
The tone is available also in the configs. If not provided by the user we should fallback to the configured one.